runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -135,7 +135,7 @@ protected ILogData getNextEntry(QueuedStreamContext context,
 
         // If maxGlobal is before the checkpoint position, throw a
         // trimmed exception
-        if (maxGlobal < context.checkpointSuccessStartAddr) {
+        if (maxGlobal < context.checkpointSuccessStartAddr && Address.isAddress(maxGlobal)) {
```

Comment:
Maybe swap the conditions. We should bail right away if it is not a valid address